src/main/java/com/gamebros/purepazaak/ai/SimpleAI.java

```diff
@@ -0,0 +1,135 @@
+package com.gamebros.purepazaak.ai;
+
+import java.util.ArrayList;
+import java.util.Optional;
+
+import com.gamebros.purepazaak.Board;
+import com.gamebros.purepazaak.Match;
+import com.gamebros.purepazaak.entity.Card;
+import com.gamebros.purepazaak.enumerable.PlayerEnum;
+
+public class SimpleAI {
+  protected Match match;
+
+  public SimpleAI(Match match) {
+    this.match = match;
+  }
+
+  public void update() {
+    // Stand if value equals 20
```

Comment:
Do these comments add value?  They should be included above the appropriate blocks if they're going to stay.